app/javascript/bundles/Coupons/components/CouponsList/CouponsListSelector.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from 'react';
+import { Table } from 'react-bootstrap';
+
+import UsedCouponsList from './UsedCouponsList.jsx';
+import ActiveCouponsList from './ActiveCouponsList.jsx';
+
+const CouponsListSelector = (props) => {
+  const {coupons, availability} = props;
+  const renderCouponsList = () => {
+    switch (availability) {
```

Comment:
you don't need this function because when a user visits the coupons page he/she always should see active coupons list